src/main.ts

```diff
@@ -36,7 +36,7 @@ async function start() {
 	probot.server.set("trust proxy", true);
 	probot.load(App);
 	probot.webhook.on("error", (err: Error) => {
-		logger.error({...err, err: JSON.stringify(err)}, "Webhook Error")
+		logger.error(err, `Webhook Error: ${JSON.stringify(err)}`)
```

Comment:
Finally found a magic formula. This will log stack trace and all elements of `err`